frontends/web/src/components/transactions/transactions.jsx

```diff
@@ -48,7 +47,7 @@ export default class Transactions extends Component {
                     <div className={style.address}>{t('transaction.details.address')}</div>
                     <div className={style.status}>{t('transaction.details.status')}</div>
                     <div className={style.fiat}>{t('transaction.details.fiat')}</div>
-                    <div className={style.currency}>{unit}</div>
+                    <div className={style.currency}>{t('transaction.details.balance')}</div>
```

Comment:
The column header now would say `Balance`, but that is wrong. The row does not show the account balance, but the transaction amount.

Reply:
can you change the tx overview to use `Amount` and `Fiat amount`?